assets/agenda/components/AgendaListItemIcons.jsx

```diff
@@ -31,32 +29,45 @@ function AgendaListItemIcons({item, planningItem, group, hideCoverages, row}) {
     });
 
     const getCoverageTootip = (coverage) => {
+        let slugline = coverage.item_slugline || coverage.slugline;
 
+        slugline = slugline ? ` '${slugline}'` : '';
         if (coverage.workflow_status === WORKFLOW_STATUS.DRAFT) {
-            return gettext('{{ type }} coverage {{ status }}', {
+            return gettext('Planned {{ type }} coverage{{ slugline }}', {
```

Comment:
not sure now if space is missing in slugline or it's with this ` | ` in it?

Reply:
it is with `|`.